plugin/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go

```diff
@@ -337,8 +337,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 			{"name": "MatchInterPodAffinity"},
 			{"name": "GeneralPredicates"},
 			{"name": "TestServiceAffinity", "argument": {"serviceAffinity" : {"labels" : ["region"]}}},
-			{"name": "TestLabelsPresence",  "argument": {"labelsPresence"  : {"labels" : ["foo"], "presence":true}}},
-			{"name": "NoVolumeNodeConflict"}
```

Comment:
@bsalamat is this ok? I want to remove my alpha feature predicate from the defaults, so it will break backwards compatibility with prior versions that are using the alpha feature.

Reply:
Make sure to put the action required label on this PR.